mujoco_ros/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ find_package(catkin REQUIRED
     image_transport
     camera_info_manager
     sensor_msgs
-    moveit_ros_planning_interface
+    # moveit_ros_planning_interface
```

Comment:
moveit_ros_planning_interface can be fully removed from CMakeLists.txt and package.xml